src/model/src/centralized_io_module.F

```diff
@@ -1388,7 +1388,7 @@ subroutine retrieve_time_dep_gridded_data (jdate, jtime, vname)
 #ifndef mpas
 !                   data_jdate = loc_jdate(f_met)
 !                   data_jtime = loc_jtime(f_met)
-!
+
 !                   if ((cio_grid_var_name(v,1) .ne. 'TSEASFC') .or.  TSEASFC_AVAIL) then
 #ifdef twoway
```

Comment:
Similar comment here, as not sure we need to comment out sections that are controlled by the "ifdef twoway" CMAQ macro.